src/sentry/crons/consumers/checkin.py

```diff
@@ -0,0 +1,61 @@
+import logging
+from typing import Mapping
+
+from arroyo.backends.kafka.consumer import KafkaPayload
+from arroyo.processing.strategies.abstract import ProcessingStrategy, ProcessingStrategyFactory
+from arroyo.processing.strategies.commit import CommitOffsets
+from arroyo.processing.strategies.run_task import RunTask
+from arroyo.types import Commit, Message, Partition
+from django.db import transaction
+
+from sentry.models import CheckInStatus, Monitor, MonitorCheckIn, MonitorStatus
+from sentry.utils import json
+from sentry.utils.dates import to_datetime
+
+logger = logging.getLogger(__name__)
+
+
+def process_message(message: Message[KafkaPayload]) -> None:
+    params = json.loads(message.payload.value)
+    current_datetime = to_datetime(params["date_updated"])
```

Comment:
@evanpurkhiser These and other places in this consumer _can_ throw in theory. We can catch all exceptions and log them to Sentry, but continue ingesting. At the moment, we halt the consumer on every exception.

What's your initial preference? Eventually we'll add a dead letter queue, but that's not needed for now.

Reply:
@wedamija might have a better opinion here. If it's just "in theory" then I don't really care what we do, assuming it will usually work.

Reply:
We should catch the exception, log and keep ingesting